engine/src/main/resources/assets/shaders/hdr_frag.glsl

```diff
@@ -31,11 +31,7 @@ void main(){
 #endif
 
 #ifdef UNCHARTED_2_TONEMAP
-    color.rgb *= exposure;
-    vec3 adjColor = uncharted2Tonemap(color.rgb);
-    vec3 whiteScale = 1.0/uncharted2Tonemap(vec3(whitePoint));
-    vec3 finalColor = adjColor*whiteScale;
-    color.rgb = finalColor;
+    color.rgb = uncharted2Tonemap(color.rgb * exposure) / uncharted2Tonemap(vec3(whitePoint));
```

Comment:
Would it be possible to still keep some nicely named variables so that noobs like me have a hope in understanding? Or a comment would do OK too.